### PR TITLE
 Update ALS o16n notebook with fixed conda env

### PR DESCRIPTION
**Pull request by loomlike**
### Description
Update ALS o16n notebook to address #1171 #1158 #900 
Now the service script does:
if the app request top 10, it uses the cached result from CosmosDB. If the app request more than top 10 movies, (e.g. top 20), then it uses the deployed model to do recommendation real time by calling model.recommendForAllUsers(20).

Run time of CosmosDB lookup vs realtime recommendation was: 0.05 sec vs 3.93 sec. 

### Related Issues
#1171 #1158 #900

### Checklist:

- [x] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [x] This PR is being made to `staging` and not `master`.

### CODE REVIEW

File: examples/05_operationalize/als_movie_o16n.ipynb [hunk 1]
```diff
@@ -112,7 +112,7 @@
    "cell_type": "markdown",
```
Comment (miguelgfierro) [inline examples/05_operationalize/als_movie_o16n.ipynb:112]: <p>I think we can remove this cell and the previous comment, in other spark notebooks we don't add it. Everything is explained in the SETUP. </p><p> </p><p>Maybe what happened is that someone was debuging the notebook and added <code>os.environ</code>  and forgot to remove before commiting. Then another person added the comment. Just a theory :-) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

**Comment by miguelgfierro** [comment]
hey Jun, let's discuss this internally

File: examples/05_operationalize/als_movie_o16n.ipynb (hunk 1 above)
Comment (loomlike) [reply examples/05_operationalize/als_movie_o16n.ipynb:112]: I'm thinking making changes in this PR.
For now the issue we discussed is, the notebook contains two separate contents 
1: using cosmosDB, 2: using AzureML for realtime prediction.

Two options are: a) Two separate sections in one notebook vs. b) Have two separate notebooks. Which one make more sense?

Comment (miguelgfierro) [reply examples/05_operationalize/als_movie_o16n.ipynb:112]: I think having two separate notebooks provides more clarity

Comment (loomlike) [reply examples/05_operationalize/als_movie_o16n.ipynb]: @miguelgfierro If we make two separate notebooks, most of the contents will be redundant.
Instead, I made changes to put the on-demand prediction part as an appendix markdown contents at the end of the notebook. Please check the changes and let me know if that looks good. I also cleaned up the env-variable cells you caught.

Comment (miguelgfierro) [reply examples/05_operationalize/als_movie_o16n.ipynb]: ok

**Review by miguelgfierro — APPROVED** [review]

**Pull request merged**